usaspending_api/common/helpers/orm_helpers.py

```diff
@@ -71,6 +71,17 @@ class FiscalYearAndQuarter(Func):
     output_field = CharField()
 
 
+class FiscalYearAndPeriod(Func):
```

Comment:
I believe this function and the annotation to support it is a duplicate of `get_fyp_or_q_notation` four functions below.

Reply:
it is! wish i could read. remove function and replaced w/ `get_fyp_or_q_notation`